### ISSUE TITLE
[Enhancement] Better Hovering Dash Icons Animations

### ISSUE THREAD

**Issue by TCMOREIRA** [micheleg/dash-to-dock#2247]
There's another extension, called Dash2Dock Animated, which implements hovering over icons animations quite better.
You should check what they're doing. Their animations are much more fluid and responsive.

**Comment by vanvugt** [comment]
The screenshot looks a little too much like macOS, and Apple is known to send cease and desist letters to anyone who copies them. So I don't think it would be a good idea. Unless that extension implements something cool that isn't in macOS?

**Comment by TCMOREIRA** [comment]
Well, I doubt it would cause any problems, it's literally just "dash to dock" but with better animations. That's the only main difference. Try it for yourself and see what I mean.

**Comment by vanvugt** [comment]
It has caused problems in the past for other projects.

Those of us who are here in a professional capacity have an extra responsibility to protect our projects and employers from litigation.

**Comment by TCMOREIRA** [comment]
Oh I didn't mean to discredit you! I understand that can happen. It might just be my ignorance, though, but, I felt that those animations were harmless to adopt as they're not so different than what we've seen already in the likes of many other popular Linux docks.

**Comment by vanvugt** [comment]
Indeed, I think small projects do get away with it. But it becomes more of a concern once you have a large user base like Ubuntu which depends on this project.

**Comment by TCMOREIRA** [comment]
Bummer! 😕
